charters/CommunicationsTeam/charter.md

```diff
@@ -0,0 +1,64 @@
+
+# Data Coordination Platform Communications Team
+
+
+## Description
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) Communications team coordinates and publishes public communications through various channels for the DCP as a whole. We work with the different groups within the DCP to ensure that each group is properly represented in external communications. We work with the HCA communications representatives if/when needed to ensure that DCP communications are consistent with and support those of the HCA. We monitor the DCP Data Portal documentation and update as needed. We develop training materials and events to teach people about the HCA DCP and how to use it.
+
+## Objective
+* Provide clear and accurate information to the public regarding DCP activities
+* Provide educational materials to help people to take advantage of DCP tools and HCA data
+* Function as the front line for communications between the DCP and legal or media entities, triaging requests or issues to the appropriate DCP or HCA leadership and possibly creating first drafts for their review (eg: press releases).
+* Support users and ensure that questions regarding DCP tools or HCA data are forwarded onto the appropriate group
+
+## In-scope
+* Update and maintain various public communication channels, for example:
+    * Social media channels
+    * Direct user communication
+    * Support applications (eg, Zendesk)
+* Develop and host educational materials  and events describing the DCP components
+* Monitor the state of our communications to ensure they are up-to-date and complete
+* Develop long-term plans for HCA DCP communication and estimate the resources needed to execute these plans
+
+### Scientific "guardrails"
```

Comment:
should be -> must be

Reply:
Good catch. Thanks! I've updated the doc and committed the change.